lib/mods/replace-get-set.js

```diff
@@ -25,25 +25,70 @@ function isDataProperty(type, propertyName) {
 }
 
 /**
- * @param {Node} node - the current ts-morph AST Node
+ * @param {Node} node - the ts-morph AST Node to evaluate
  *
- * @return {boolean} whether the node is a global get
+ * @return {boolean} whether the node is a global get/set
  */
-function isGlobalGet(node) {
+function isGlobalPropertyAccess(node) {
   return nodeTypes.isPropertyAccessExpression(node) &&
     !nodeTypes.isCallExpression(node.getParent()) &&
     isGlobal(node.getExpression());
 }
 
 /**
- * @param {Node} node - the current ts-morph AST Node
+ * @param {Node} node - the ts-morph AST Node to evaluate
  *
  * @return {boolean} whether the node is a global set
  */
-function isGlobalSet(node) {
+function isSet(node) {
   return nodeTypes.isBinaryExpression(node) &&
-    node.getOperatorToken().getText() === '=' &&
-    isGlobalGet(node.getLeft());
+    node.getOperatorToken().getText() === '=';
+}
+
+/**
+ * Checks if node in question is a non-property access window
+ * identifier, by eliminating other possible types of globals
+ * the identifier could be
+ *
+ * @param {Node} node - the ts-morph AST Node to evaluate
+ *
+ * @return {boolean} whether the node is a window global
+ */
+function isWindowGlobal(node) {
+  // filter non-globals
+  const symbol = node.getSymbol();
```

Comment:
Can we dedupe this code with isGlobal from originals-helper.js?

Reply:
I looked into this we actually can't trivially, since `isGlobal` is checking if the type of the node is declared in TypeScript, whereas for window we want to see if the symbol itself was declared in TypeScript